deploy/backplane/srep/10-srep-admins-project.ClusterRole.yml

```diff
@@ -19,6 +19,13 @@ rules:
   - pods/eviction
   verbs:
   - create
+# SRE can portforward pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/portforward
+  verbs:
+  - create
```

Comment:
do we want to limit this so that we allow port forwarding in only openshift-monitoring ns for SREP, keeping in mind least privilege access? Otherwise this will get applied to all ns in the the subject permission.

Reply:
in 10-cssre.SubjectPermission.yml, we limit to openshift-logging namespace.